sharktank/sharktank/utils/testing.py

```diff
@@ -169,7 +215,15 @@ def print_stats(label, t):
         for name in ref_f.keys():
             actual = actual_f.get_tensor(name)
             ref = ref_f.get_tensor(name)
-            torch.testing.assert_close(actual, ref, msg=name)
+            try:
```

Comment:
isn't this redundant to what torch.testing.assert_close already does?

Reply:
If `fail_fast` is `False` then we will keep going and print the failures. I did this to allow for a full report that shows all discrepancies.